client/src/components/Onboarding.js

```diff
@@ -68,9 +68,44 @@ function ProgressMobileStepper({activeStep}) {
     );
 }
 
-function Onboarding({children,title,activeStep,path}) {
+function Onboarding({children,title,activeStep,path,url}) {
 
     const classes = useStyles();
+    
+    const history = useHistory();
+
+    const [urlStore, setUrlStore] = React.useState("");
+
+    const handleContinueClick = () => {
+        let status;
+        if (activeStep === 1){
+            if (url === "") return alert('empty url');
+            fetch("/user/"+url+"/event-type", {
+                method: "GET",
+                headers: {
+                    "Content-Type": "application/json"
+                }
+            })
+                .then(res => {
+                    status = res.status;
+                    if (status < 500) return res.json();
+                    else throw Error("Server error");
+                })
+                .then(res => {
+                    if (status === 200) {
+                        if (res.unique === "true") {
```

Comment:
this is a small thing but why don't we return a boolean instead of a string?

Reply:
I was trying to do that but the boolean in python and javascript is different, and when I test by `(res.unique === true)` it doesn't work.
I guess it would work if I pass `{unique: True}` as boolean in BE would work for  `(res.unique === true)`